src/NServiceBus.Core.Tests/Routing/UnicastSendRouterTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace NServiceBus.Core.Tests.Routing
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NServiceBus.Routing;
+    using Transport;
+    using Unicast.Messages;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class UnicastSendRouterTests
+    {
+        UnicastSendRouter router;
+        MessageMetadataRegistry metadataRegistry;
+        UnicastRoutingTable routingTable;
+        EndpointInstances endpointInstances;
+        TransportAddresses transportAddresses;
+
+        [Test]
+        public async Task When_routing_command_to_logical_endpoint_without_configured_instances_should_route_to_a_single_destination()
+        {
+            var logicalEndpointName = "Sales";
+            routingTable.RouteToEndpoint(typeof(Command), logicalEndpointName);
+
+            var routes = await router.Route(typeof(Command), new DistributionPolicy(), new ContextBag());
+
+            Assert.AreEqual(1, routes.Count());
```

Comment:
that doesn't really make sense here? I inlined the destination variable though (on the next line).